portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ActiveIndividuals.testcase

```diff
@@ -166,9 +166,45 @@ definition {
 	@ignore = "true"
 	@priority = "5"
 	test GraphShowsActivitiesFromTodayWhenSwitchedTo24HrFilter {
+		ACUtils.createDMAndAddToPage();
+
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+
+		JSONUser.addUser(
+			userEmailAddress = "ac@liferay.com",
+			userFirstName = "ac",
+			userLastName = "ac",
+			userScreenName = "ac");
+
+		ACDXPSettings.syncNewContactData();
+
+		User.logoutAndLoginPG(
+			userLoginEmailAddress = "ac@liferay.com",
+			userLoginFullName = "ac ac");
+
+		for (var n : list "1,2,3,4") {
+			Navigator.gotoSitePage(
+				pageName = "AC Page",
+				siteName = "Site Name");
 
-		// TODO LRAC-8139 GraphShowsActivitiesFromTodayWhenSwitchedTo24HrFilter pending implementation
+			DMNavigator.gotoDocumentCP(dmDocumentTitle = "DM AC Title");
+		}
+
+		ACUtils.closeAllSessions();
+
+		Pause(locator1 = "30000");
+
+		ACUtils.launchAC();
+
+		ACProperties.switchProperty(propertyName = "Liferay DXP Combined Property");
+
+		ACNavigation.goToIndividuals();
 
+		ACTimeFilter.clickTimeFilterButton();
+
+		ACTimeFilter.setLast24Hours();
+
+		AssertElementPresent(locator1 = "ACIndividualsDashboard#ACTIVE_INDIVIDUAL_CHART");
```

Comment:
Hey @ViniLopes20 , I was taking a look at this path and it is only checking any option selected on the time filter dropdown. As well as, in this test we should validate if the graph shows activities when switching to the last 24 hours... So we should investigate for example how to validate a blue bar that is probably shown based on the DXP activities. ;)

Reply:
For this, please, take a look at these tests created by Calvin, I think he was validating something very similar to what you are trying to validate here:

```
ActiveIndividuals#CanSetDataWithCustomRangeOnIndividualActivitiesCard
ActiveIndividuals#CanSetEmptyDataCustomRangeOnIndividualActivitiesCard
```